public/app/features/expressions/components/Math.tsx

```diff
@@ -12,24 +14,143 @@ interface Props {
+        </div>
+        <header className={styles.documentationHeader}>Available Math functions</header>
+        <div className={styles.documentationFunctions}>
+          <DocumentedFunction
+            name="abs"
+            description="returns the absolute value of its argument which can be a number or a series"
+          />
+          <DocumentedFunction
+            name="is_inf"
+            description="takes a number or a series and returns 1 for Inf values (negative or positive) and 0 for other values"
+          />
+          <DocumentedFunction
+            name="is_nan"
+            description="takes a number or a series and returns 1 for NaN values and 0 for other values"
+          />
+          <DocumentedFunction
+            name="is_null"
+            description="takes a number or a series and returns 1 for null values and 0 for other values"
+          />
+          <DocumentedFunction
+            name="is_number"
+            description="takes a number or a series and returns 1 for all real number values and 0 for non-number"
+          />
+          <DocumentedFunction
+            name="log"
+            description="returns the natural logarithm of of its argument which can be a number or a series"
+          />
+          <DocumentedFunction
+            name="inf, infn, nan, and null"
+            description="The inf, infn, nan, and null functions all return a single value of the name"
+          />
+          <DocumentedFunction name="round" description="returns a rounded integer value" />
+          <DocumentedFunction name="ceil" description="rounds the number up to the nearest integer value" />
+          <DocumentedFunction name="floor" description="rounds the number down to the nearest integer value" />
+        </div>
+        <div>
+          See our additional documentation on{' '}
+          <a
+            className={styles.documentationLink}
+            target="_blank"
+            href="https://grafana.com/docs/grafana/latest/panels/query-a-data-source/use-expressions-to-manipulate-data/about-expressions/#math"
+            rel="noreferrer"
+          >
+            <Icon size="xs" name="external-link-alt" /> Math expressions
+          </a>
+          .
+        </div>
+      </div>
+    </Stack>
   );
 };
+
+interface DocumentedFunctionProps {
+  name: string;
+  description: React.ReactNode;
+}
+const DocumentedFunction = ({ name, description }: DocumentedFunctionProps) => {
+  const styles = useStyles2(getDocumentedFunctionStyles);
+
+  return (
+    <>
+      <span className={styles.name}>{name}</span>
+      <span className={styles.description}>{description}</span>
+    </>
+  );
+};
+
+const getStyles = (theme: GrafanaTheme2, showHelp?: boolean) => ({
+  documentationHeader: css`
+    font-size: ${theme.typography.h5.fontSize};
+    font-weight: ${theme.typography.h5.fontWeight};
+  `,
+  documentationLink: css`
+    color: ${theme.v1.colors.linkExternal};
```

Comment:
Isn't `theme.v1` something we should avoid as deprecated?

Reply:
Right good point, I'm still a bit confused because everything that's not under `v1` is marked as `beta` 😓